packages/react-next/src/components/TeachingBubble/TeachingBubble.base.tsx

```diff
@@ -1,78 +1,59 @@
 import * as React from 'react';
-import { initializeComponentRef, classNamesFunction } from '../../Utilities';
+import { classNamesFunction } from '../../Utilities';
 import { TeachingBubbleContent } from './TeachingBubbleContent';
 import {
+  ITeachingBubble,
   ITeachingBubbleProps,
   ITeachingBubbleStyleProps,
   ITeachingBubbleStyles,
   ITeachingBubbleSubComponentStyles,
 } from './TeachingBubble.types';
 import { Callout, ICalloutProps } from '../../Callout';
 import { DirectionalHint } from '../../common/DirectionalHint';
+import { useMergedRefs } from '@uifabric/react-hooks';
 
-const getClassNames = classNamesFunction<ITeachingBubbleStyleProps, ITeachingBubbleStyles>();
-
-export interface ITeachingBubbleState {
-  isTeachingBubbleVisible?: boolean;
-}
-
-export class TeachingBubbleBase extends React.Component<ITeachingBubbleProps, ITeachingBubbleState> {
-  public static defaultProps = {
-    /**
-     * Default calloutProps is deprecated in favor of private `_defaultCalloutProps`.
-     * Remove in next release.
-     * @deprecated In favor of private `_defaultCalloutProps`.
-     */
-    // eslint-disable-next-line deprecation/deprecation
-    calloutProps: {
-      beakWidth: 16,
-      gapSpace: 0,
-      setInitialFocus: true,
-      doNotLayer: false,
-      directionalHint: DirectionalHint.rightCenter,
-    },
-  };
-
-  public rootElement = React.createRef<HTMLDivElement>();
-  private _defaultCalloutProps: ICalloutProps;
-
-  // Constructor
-  constructor(props: ITeachingBubbleProps) {
-    super(props);
-
-    initializeComponentRef(this);
-    this.state = {};
+const COMPONENT_NAME = 'TeachingBubble';
+const defaultCalloutProps: ICalloutProps = {
+  beakWidth: 16,
+  gapSpace: 0,
+  setInitialFocus: true,
+  doNotLayer: false,
+  directionalHint: DirectionalHint.rightCenter,
+};
 
-    this._defaultCalloutProps = {
-      beakWidth: 16,
-      gapSpace: 0,
-      setInitialFocus: true,
-      doNotLayer: false,
-      directionalHint: DirectionalHint.rightCenter,
-    };
-  }
-
-  public focus(): void {
-    if (this.rootElement.current) {
-      this.rootElement.current.focus();
-    }
-  }
+const getClassNames = classNamesFunction<ITeachingBubbleStyleProps, ITeachingBubbleStyles>();
+const useComponentRef = (
+  componentRef: React.Ref<ITeachingBubble> | undefined,
+  rootElementRef: React.RefObject<HTMLDivElement>,
+) => {
+  React.useImperativeHandle(
+    componentRef,
+    () => ({
+      focus: () => rootElementRef.current?.focus(),
+    }),
+    [rootElementRef],
+  );
+};
 
-  public render(): JSX.Element {
+export const TeachingBubbleBase = React.forwardRef(
+  (props: ITeachingBubbleProps, forwardedRef: React.Ref<HTMLDivElement>) => {
+    const rootElementRef = React.useRef<HTMLDivElement>(null);
```

Comment:
In the previous implementation, this was exposed as a public member for some reason, which if you're directly translating means it should go on the `useComponentRef`. It's probably okay to leave that out but it might be worth noting in RELEASE_NOTES.md breaking changes.